### PR TITLE
Sp 442 add new model fused cylindrical in kognic io

### PR DESCRIPTION
**Pull request by luca-caltagirone**
Added support for Fused Cylindrical camera model

### CODE REVIEW

**Review by olinoskar — APPROVED** [review]
LGTM!
Don't forget to update the changelog in the README!

File: kognic-io/kognic/io/model/calibration/camera/fused_cylindrical_calibration.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+class CutAngles(BaseSerializer):
+    upper: float
+    lower: float
+
+
+class FusedCylindricalCalibration(BaseCameraCalibration):
+    calibration_type = CalibrationType.FUSEDCYLINDRICAL.value
+    cut_angles_degree: CutAngles
+    vertical_fov_degree: Optional[float] = 90
```
Comment (olinoskar) [inline kognic-io/kognic/io/model/calibration/camera/fused_cylindrical_calibration.py:15]: Can these be `None` or are they optional in the sense that they have a default value?

Comment (olinoskar) [reply kognic-io/kognic/io/model/calibration/camera/fused_cylindrical_calibration.py:15]: Also, can we rename it to vertical_fov_degree**s** instead? And the same with horizontal_fov_degree

Comment (luca-caltagirone) [reply kognic-io/kognic/io/model/calibration/camera/fused_cylindrical_calibration.py]: Yes, exactly. They have default values but the client could override them in the future if needed

**Pull request merged**